src/Browscap/Data/Expander.php

```diff
@@ -71,6 +72,7 @@ private function resetPatternId() : void
             'useragent' => '',
             'platform' => '',
             'device' => '',
+            'browser' => '',
```

Comment:
I'm not sure this is necessary to have the browser (or the engine) in the pattern id.

They're not used in the `getPatternId` method, and not included in coverage reports.

If you really do think we should add them (because you want to expand coverage to process the browser and engine files), then you should make sure you add "engine" here too (since you're setting it below).

Reply:
fixed